vue/src/components/LayerSelection.vue

```diff
@@ -1,18 +1,55 @@
 <script setup lang="ts">
-import { computed } from "vue";
+import { computed, onMounted } from "vue";
 import { ApiService } from "../client";
 import { state } from "../store";
+import { useRoute } from 'vue-router';
 
 
 
 const scoringApp = computed(()=> ApiService.getApiPrefix().includes('scoring'));
+const route = useRoute();
+const proposals = computed(() => route.path.includes('proposals'));
 
 
+const toggleGroundTruth = (id: string) => {
+  if (state.filters.drawObservations?.includes('groundtruth') || state.filters.drawSiteOutline?.includes('groundtruth')) {
+        if (state.openedModelRuns) {
+          state.openedModelRuns.add(id);
+        }
+        const configuration_id = Array.from(state.openedModelRuns);
+        state.filters = { ...state.filters, configuration_id};
+      }
+      else if (!state.filters.drawObservations?.includes('groundtruth') && !state.filters.drawSiteOutline?.includes('groundtruth')) {
+        if (state.openedModelRuns) {
+          if (state.openedModelRuns.has(id)) {
+            state.openedModelRuns.delete(id);
+          }
```

Comment:
```suggestion
          state.openedModelRuns.delete(id);
```

Calling `Set.has()` before `Set.delete()` isn't necessary - [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/delete)